babel.config.js

```diff
@@ -59,12 +71,13 @@ module.exports = function getBabelConfig(api) {
         version: '^7.4.4',
       },
     ],
-    [
-      'babel-plugin-transform-react-remove-prop-types',
-      {
-        mode: 'unsafe-wrap',
-      },
-    ],
+    // [
```

Comment:
Yes, it can save bandwidth in production by removing the prop-types. Without it, the prop-types are always bundled.

Reply:
I'm only waiting this comment to addressed to approve. We have lots of prop-types and if developers don't remove them it will be included in the production bundle. Everything else seems good.